modules/installation-configuration-parameters.adoc

```diff
@@ -2736,107 +2730,73 @@ The list of datacenters must match the list of datacenters specified in the `vce
     failureDomains:
       topology:
         datastore:
+ifdef::vsphere[]
+|Specifies the path to a vSphere datastore that stores virtual machines files for a failure domain. You must apply the `datastore` role to the vSphere vCenter datastore location.
+endif::vsphere[]
+ifdef::agent[]
 |The path to the vSphere datastore that holds virtual machine files, templates, and ISO images.
```

Comment:
Hi @rvanderp3 and @rwsu . Is the descrption for `datastore` OK for the Agent-based Installer and vSphere?

Reply:
The only part I didn't understand was _"apply the `datastore` role to the vSphere vCenter datastore location."_ Is that a configuration setting inside vSphere? 

The rest seems to line up with the official product documentation: https://docs.openshift.com/container-platform/4.15/installing/installing_vsphere/upi/installing-vsphere.html#installation-vsphere-config-yaml_installing-vsphere

`The path to the vSphere datastore that holds virtual machine files, templates, and ISO images.`

Reply:
Is there a reason the description for datastore is different between the two preview links?

Reply:
Hi @rvanderp3 . Does the `datastore` parameter description for the Agent-based Installer also apply to vSphere core? Would be great to remove the conditional statement as two descrption variants exist for this parameter.

Reply:
> Specifies the path to a vSphere datastore that stores virtual machines files for a failure domain. You must apply the `datastore` role to the vSphere vCenter datastore location

would be my preferred description.  the `datastore role` refers to the privileges required by the account OpenShift uses to communicate with vCenter.  wdyt @jcpowermac ?

Reply:
I agree with @rvanderp3